Misc/NEWS.d/next/Documentation/2024-11-01-12-35-32.gh-issue-126273.t_qOoH.rst

```diff
@@ -0,0 +1 @@
+Added docstrings to certain xml.etree methods.
```

Comment:
```suggestion
Added docstrings to certain :class:`xml.etree.ElementTree` methods.
```

Reply:
My old nemesis rst strikes again... thanks for tidying this up!